packages/core/styles/components/_dialog.scss

```
@@ -11,8 +11,8 @@ dialog {
 
   box-shadow: var(--ix-shadow);
 
-  max-width: 36rem;
-  min-width: 24rem;
+  max-width: 28rem;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)